app/models/like.rb

```diff
@@ -1,4 +1,10 @@
 class Like < ApplicationRecord
   belongs_to :user
   belongs_to :post
+
+  after_save :likes_counter
```

Comment:
- [ ] Seeing as the ERD diagram specifies the column name to be `author_id` instead of `user_id`, the `belongs_to` relationship between the `likes` and `author` would be correctly implemented like this 👇 
```ruby
belongs_to :author, class_name: 'User'
```
Please, you will have to rename the `user_id` column on your `likes` table to `author_id` in order to correctly follow the specifications on the ERD diagram